.github/workflows/update-kopia-image.yaml

```diff
@@ -0,0 +1,41 @@
+name: Update Kopia Image
+
+on: 
+  workflow_dispatch:
+    inputs:
+      commitId:
+        description: "Commit ID"
+        required: true
+        type: string
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    permissions:
+      packages: write
+    steps:
+    - uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    - uses: actions/checkout@v2
+    - run: make update-kopia-image COMMIT_ID=${{ github.event.inputs.commitId }}
+  updateref:
+    runs-on: ubuntu-20.04
+    needs: [build]
+    steps:
+    - uses: actions/checkout@v2
+    - run: make update-image-reference KOPIA_IMAGE_TAG="ghcr.io/kanisterio/kopia:alpine-${{ github.event.inputs.commitId }}"
+    - uses: peter-evans/create-pull-request@v4
```

Comment:
I know this is the only one with a decent amount of stars in the marketplace but we should make sure it's safe to use.